aiohttp/web_request.py

```diff
@@ -494,6 +497,47 @@ def if_unmodified_since(self) -> Optional[datetime.datetime]:
         """
         return self._http_date(self.headers.get(hdrs.IF_UNMODIFIED_SINCE))
 
+    @staticmethod
+    def _etag_values(etag_header: str) -> Iterator[ETag]:
+        """Extract `ETag` objects from raw header."""
+        if etag_header == ETAG_ANY:
+            yield ETag(
+                is_weak=False,
+                value=ETAG_ANY,
+            )
+        else:
+            for match in LIST_QUOTED_ETAG_RE.finditer(etag_header):
+                is_weak, quoted_value = match.group(2), match.group(3)
```

Comment:
You can pass several arguments to `group()`:
```suggestion
                is_weak, quoted_value = match.group(2, 3)
```

Reply:
That's a lot better, thanks!